### PR TITLE
Move sample listings creation out of contracts migration scripts #1113

### PR DESCRIPTION
**Pull request by nihar97**
First pull request? Read our [guide to contributing](http://docs.originprotocol.com/#contributing)

### Description:

Please explain the changes you made here:

- A description of the problem you're trying to solve
- An overview of the suggested solution
- If the feature changes current behavior, reasons why your solution is better

### Checklist:

- [ ] Test your work and double-check to confirm that you didn't break anything
- [x] Wrap any displayed ETH addresses with [`formattedAddress`](https://github.com/OriginProtocol/origin/blob/master/origin-dapp/src/utils/user.js#L15-L17)
- [x] Wrap any new text/strings for translation
- [x] Map any new environment variables with a default value in the Webpack config
- [x] Update any relevant READMEs and [docs](https://github.com/OriginProtocol/origin/tree/master/origin-docs)

This code should change the place where sample data is populated as discussed in the issue. However, in an effort to not break abstraction barriers I have to use the marketplace resource. Since this resource uses the ES6 import statement, the system is broken by this update. Once we figure out how to babelify the serve.js file it should work as expected.

### CODE REVIEW

File: origin-js/scripts/serve.js [hunk 1]
```diff
@@ -17,4 +19,141 @@ const start = async () => {
...
+    ipfsService,
+    affiliate: accounts[3],
+    arbitrator: accounts[4],
+    store,
+    blockEpoch: await web3.eth.getBlockNumber()
+  })
+
+  //hawai-house
+  await marketplace.createListing({
+    listingType: 'unit',
```
Comment (franckc) [inline origin-js/scripts/serve.js]: As opposed to hardcoding the listing data, how about using the [sample fixtures](https://github.com/OriginProtocol/origin/tree/master/origin-js/test/fixtures) ?

File: origin-js/scripts/serve.js [hunk 2]
```diff
@@ -17,4 +19,141 @@ const start = async () => {
...
+    ipfsApiPort: '5002',
+    ipfsGatewayPort: '8080',
+    ipfsGatewayProtocol: 'http'
+  })
+  const store = new StoreMock()
+  store.set(
+    'notification_subscription_start',
+    new Date().getTime()
+  )
+  marketplace = new Marketplace({
```
Comment (franckc) [inline origin-js/scripts/serve.js]: As opposed to instantiating directly a Marketplace object, how about creating an Origin object which is the base for using the origin-js APIs ?
See an example [here](https://github.com/OriginProtocol/origin/blob/master/origin-tests/test/1-create-listing.js#L19).
Also see our (still work in progress) documentation [here](https://docs.originprotocol.com/#origin-object).

In order to import Origin, could you try [this](https://github.com/OriginProtocol/origin/blob/master/origin-tests/test/1-create-listing.js#L8) and see if that works ?

Comment (nihar97) [reply origin-js/scripts/serve.js]: Yes. I will try that. I initially did use an Origin object but I ran into the same problem of imports.

Comment (nihar97) [reply origin-js/scripts/serve.js]: I changed the code to use and Origin object and also to use sample fixtures. However, that style of importing origin still doesn't work

Comment (franckc) [reply origin-js/scripts/serve.js]: hmm... what error do you get exactly ?

Comment (nihar97) [reply origin-js/scripts/serve.js]: ` /app/origin-js/src/index.js:1
origin                  | 
(function (exports, require, module, __filename, __dirname) { import ContractService from './services/contract-service'
origin                  |                                                                      ^^^^^^^^^^^^^^^
origin                  |
origin                  | SyntaxError: Unexpected identifier`

Comment (tomlinton) [reply origin-js/scripts/serve.js]: You'll have to use `babel-node` to get support for ES6. E.g. `babel-node scripts/serve.js --preset @babel/env`

Comment (nihar97) [reply origin-js/scripts/serve.js]: hmm should I add babel-node to the dependencies?

Comment (franckc) [reply origin-js/scripts/serve.js]: Sgtm. Please try adding babel-node and see if that helps...

**Comment by franckc** [comment]
This issue has been fixed as part of DApp2 rewrite. Closing.

**Pull request closed without merging**